src/main/java/uk/gov/hmcts/reform/idam/util/IdamTokenGenerator.java

```diff
@@ -22,7 +22,7 @@ public class IdamTokenGenerator {
     private final IdamClient idamClient;
     private final ParameterResolver parameterResolver;
 
-    private String idamClientToken;
+    private String idamClientToken = "token";
```

Comment:
You could create an empty string instead - StringUtils.EMPTY

Reply:
I could, but I prefer to set it to something so that integration test can check that it is being set to "Authorization: Bearer token", rather than "Authorization: Bearer".